pkg/alertmanager/README.md

```diff
@@ -0,0 +1,108 @@
+## Alertmanager Package
+
+The alertmanager package and CRDs together make up the alertmanager integration in the prometheus operator. Here is a
+quick overview of the overall package:
+
+![pkg/alertmanager and crds](../../Documentation/img/am_readme.png)
+
+### CRDs
+
+Prometheus operator supports two CRDs related to Alertmanager.
+
+- Alertmanager CRD: Defines the alertmanager statefulset.
+- AlertmanagerConfig CRD: Deines the config for the alertmanager instances.
+
+The Alertmanager CRD is in [v1](https://github.com/prometheus-operator/prometheus-operator/blob/main/README.md#project-status).
+while AlertmanagerConfig CRD is currently in [beta](https://github.com/prometheus-operator/prometheus-operator/blob/main/README.md#project-status).
```

Comment:
in practice we have 2 API versions for the AlertmanagerConfig CRD: v1alpha1 and v1beta1 (the former being the default stored version).